assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/data/useInitialState.ts

```diff
@@ -14,6 +18,12 @@ import { ticketFieldsToUse } from '../utils/constants';
 const useInitialState = ({ ticketId }: BaseProps): StateInitializer => {
 	const { getRelations } = useRelations();
 
+	const allPriceTypes = usePriceTypes();
+	const [basePriceType] = allPriceTypes.filter(({ isBasePrice }) => isBasePrice === true);
```

Comment:
There is a predicate `isBasePrice` in `@sharedEntities/priceTypes/predicates/selectionPredicates` that you can use here.

Reply:
have used, thx